generated/nidaqmx/_task_modules/in_stream.py

```diff
@@ -330,22 +331,23 @@ def input_onbrd_buf_size(self):
         return val
 
     @property
-    def logging_file_path(self):
+    def logging_file_path(self) -> pathlib.PurePath:
         """
-        str: Specifies the path to the TDMS file to which you want to
-            log data.  If the file path is changed while the task is
-            running, this takes effect on the next sample interval (if
-            Logging.SampsPerFile has been set) or when DAQmx Start New
-            File is called. New file paths can be specified by ending
-            with "\\" or "/". Files created after specifying a new file
-            path retain the same name and numbering sequence.
+        pathlib.PurePath: Specifies the path to the TDMS file to which
+            you want to log data.  If the file path is changed while the
+            task is running, this takes effect on the next sample
+            interval (if Logging.SampsPerFile has been set) or when
+            DAQmx Start New File is called. New file paths can be
+            specified by ending with "\\" or "/". Files created after
+            specifying a new file path retain the same name and
+            numbering sequence.
         """
 
         val = self._interpreter.get_read_attribute_string(self._handle, 0x2ec4)
         return val
```

Comment:
The interpreter returns a `str`, so this layer is responsible for converting to `pathlib.Path`, just like when we return an enum.

Reply:
Noted, it's changed according to suggestion.